Connectors/src/MySqlEFCore/appsettings.Development.json

```diff
@@ -22,11 +22,11 @@
           "ConnectionString": "Server=localhost;Database=steeltoe;User=steeltoe;Password=steeltoe;Connection Timeout=15"
         }
         /*
-        "myMySqlServiceOne": {
+        "sampleMySqlServiceOne": {
```

Comment:
What do you think about calling this one `sampleMySqlService` (to match the others) and calling the second one in this sample `secondMySqlService` or something along those lines, so that we can at least be setup to share one instance if we go that route?

Reply:
It could easily be done here, but it would complicate the contents of `appsettings.Development.json`: The first one needs to move from `Default` to a named entry. Also, the connection strings are different in all three cases, to make it easier to spot which one is being used.

Reply:
Nevermind, this doesn't matter